.github/workflows/on-pr-submit.yml

```diff
@@ -43,9 +45,21 @@ jobs:
     - name: Generate Manifests
       run: VERBOSE="true" make gen-manifest
     - name: Setup Minikube
-      run: tests/setup.sh v1.20.0
+      run: tests/setup.sh $KUBE_VERSION
     - name: Run tests
-      run: tests/travis-test.sh v1.20.0
+      run: tests/travis-test.sh $KUBE_VERSION
     - name: Cleanup
-      run: tests/cleanup.sh v1.20.0
+      run: tests/cleanup.sh $KUBE_VERSION
 
+      # Deploy KaDalu via Helm and run tests if PR address Helm charts
+    - name: Setup Minikube for Helm Charts
+      if: "contains(github.event.head_commit.message, '[helm]')"
+      run: tests/setup.sh $KUBE_VERSION
+    - name: Run tests
+      if: "contains(github.event.head_commit.message, '[helm]')"
+      run: tests/travis-test.sh $KUBE_VERSION
+      env:
+        INSTALL_TYPE: helm
```

Comment:
this may be needed in above step too.

Reply:
- in the underlying script (minikube.sh) i didn't notice any usage of `kubectl apply` command so INSTALL_TYPE for this specific run (Run tests) is enough
- the INSTALL_TYPE value if not initialised is set to `default` in `minikube.sh` so installation of resource from manifests takes place